<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анус бомжа</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="ImageArea" name="imagearea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>640</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>1024</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="points">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuDifficulty">
     <property name="title">
      <string>Difficulty</string>
     </property>
     <addaction name="actionEasy"/>
     <addaction name="actionNormal"/>
     <addaction name="actionHard"/>
    </widget>
    <widget class="QMenu" name="menuSize">
     <property name="title">
      <string>Size</string>
     </property>
     <addaction name="action10x10"/>
     <addaction name="action20x20"/>
     <addaction name="action30x30"/>
    </widget>
    <addaction name="menuDifficulty"/>
    <addaction name="menuSize"/>
    <addaction name="actionSound"/>
    <addaction name="actionHints"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEasy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
  <action name="action10x10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10x10</string>
   </property>
  </action>
  <action name="action20x20">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20x20</string>
   </property>
  </action>
  <action name="action30x30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30x30</string>
   </property>
  </action>
  <action name="actionHints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hints</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageArea</class>
   <extends>QWidget</extends>
   <header>imagearea.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
